src/configs/env.js

```diff
@@ -0,0 +1,117 @@
+import path from 'path';
+
+import { Joi } from 'express-validation';
```

Comment:
Joi is a package which is the most powerful schema description language and data validator for JavaScript.

FYI: [JOI](https://www.npmjs.com/package/joi#the-most-powerful-schema-description-language-and-data-validator-for-javascript)